ipatests/test_integration/test_replica_promotion.py

```diff
@@ -925,3 +925,21 @@ def test_hidden_replica_backup_and_restore(self):
         # FIXME: restore turns hidden replica into enabled replica
         self._check_config([self.master, self.replicas[0]])
         self._check_server_role(self.replicas[0], 'enabled')
+
+    def test_hidden_replica_automatic_crl(self):
+        """Exercises if automatic CRL configuration works with
+           hidden replica.
+        """
+        self._check_server_role(self.replicas[0], 'hidden')
```

Comment:
The previous test did a backup/restore on replicas[0], and this operation turns the hidden replica into a enabled replica. The test needs to hide the replicas[0] first.